ui/00_message/common/item_info.xml

```diff
@@ -60,6 +59,8 @@ specified status effect.</translation>
   <item id="46">
     <original>ドリード城離れにある《宝物殿》の鍵
 かつては貴重資料が集められていた</original>
+    <translation>A key that opens the Treasury in Dreed Castle.
+      Various tresures were gathered there.</translation>
```

Comment:
When adding new lines for a multi-line message, do not add spaces before the line. That will show up in-game.